docs/plugins/graphback-schema.md

```diff
@@ -21,24 +21,14 @@ npm install @graphback/codegen-schema
 
 ### Configuration
 
-`SchemaCRUDPlugin` can be configured to write your generated schema to a file:
+By default `SchemaCRUDPlugin` will not generate any schema file, instead creating an in-memory schema. You might sometimes need the schema file for compatibility with other libraries or frameworks, so Graphback allows you to automate the persistence of the schema to a file in your project:
 
 
 ```ts
-  /**
-   * Output format for schema
-   */
-  format: 'ts' | 'js' | 'graphql',
-
   /**
    * RelativePath for the output files created by generator
    */
   outputPath: string
```

Comment:
outputFile?

Reply:
But you can just specify the folder also, which is not a file. Path is still technically correct for both scenarios.